<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设备通信管理器</string>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_device">
    <property name="title">
     <string>设备</string>
    </property>
    <addaction name="action_connect_all"/>
    <addaction name="action_disconnect_all"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_tools">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="action_clear_log"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menu_device"/>
   <addaction name="menu_tools"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolbar">
   <property name="windowTitle">
    <string>工具栏</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_connect_all"/>
   <addaction name="action_disconnect_all"/>
   <addaction name="separator"/>
   <addaction name="action_clear_log"/>
  </widget>
  <action name="action_connect_all">
   <property name="text">
    <string>连接所有设备</string>
   </property>
   <property name="toolTip">
    <string>连接所有已配置的设备</string>
   </property>
  </action>
  <action name="action_disconnect_all">
   <property name="text">
    <string>断开所有设备</string>
   </property>
   <property name="toolTip">
    <string>断开所有已连接的设备</string>
   </property>
  </action>
  <action name="action_clear_log">
   <property name="text">
    <string>清除日志</string>
   </property>
   <property name="toolTip">
    <string>清除日志窗口的内容</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>关于</string>
   </property>
   <property name="toolTip">
    <string>显示关于对话框</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui> 